presto-docs/src/main/sphinx/connector/mysql.rst

```diff
@@ -66,6 +66,9 @@ Property Name                                      Description
 
 ``case-insensitive-name-matching.cache-ttl``       Duration for which remote dataset and table names will be
                                                    cached. Set to ``0ms`` to disable the cache.                         ``1m``
+
+``mixed-case-support``                             Enable mixed case identifier support for schema and table names      ``false``
+                                                   for the connector
```

Comment:
```suggestion
``mixed-case-support``                             Enable mixed case identifier support for schema and table names      ``false``
                                                   for the connector. When not enabled identifiers for schema and table names  are converted to lowercase.
```

Reply:
Not sure about others, to me word `mixed case` is unheard of. It may be good to include, what do we imply by mixed case support.

Reply:
Sure, I am open for suggestion here?